docs/content/en/docs/implementing/slo/_index.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Analisis
-description: Understand Analysis in Keptn and how to use them
-weight: 150
-cascade:
-  type: docs
-  currentversion: main
+title: Define SLOs/SLIs with Analyses
+description: Understand Analyses in Keptn and how to use them
+weight: 91
 ---
 
 The Keptn Metrics Operator implements an SLO/SLI feature set inspired by Keptn v1 under the name of Analysis.
-With an Analysis Definition you can specify multiple Service Level Objectives that will be evaluated in your Analysis.  
-At the end of the Analysis the status will return whether your objective failed or passed.
+With an Analysis Definition you can specify multiple Service Level Objectives (SLOs) that will be evaluated in your Analysis.
+At the end of the Analysis the status returns whether your objective failed or passed.
 
-This data can be seen as Prometheus metrics and can be displayed on Grafana.
+The Analysis result is exposed as an OpenTelemetry metric and can be displayed on dashboard tools, such as Grafana.
 
-Keptn v1 users may use [SLO](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/slo_converter.md#slo-converter) and [SLI converters](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/sli_converter.md#sli-converter) to migrate towards Keptn Analysis. 
+Keptn v1 users may use converters for
+[SLOs](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/slo_converter.md#slo-converter)
+and [SLIs](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/converter/sli_converter.md#sli-converter)
```

Comment:
we could use release please comment annotations here and then later, a release tag instead of main branch. same for the other github links in this PR

Reply:
we agreed with Meg that in a separate pr she  will move those readmes in .md files in the documentation